modules/test/playwright/pages/account-admin-web/AccountsPage.ts

```diff
@@ -82,6 +109,15 @@ export class AccountsPage {
 				strictEqual
 			);
 		};
+		this.accountsTableRowCheckBox = async (name: string) => {
```

Comment:
I think all these stuff starting with 'a' should be moved at the beginning. I can see that most of them were not created by you, so let's see what Stefano says.